README.md

```diff
@@ -1,2 +1,214 @@
 # bclconvert
-Workflow to process a run directory through bclconvert on hpc
+
+Workflow to produce FASTQ files from an Illumina instrument's run directory using DRAGEN bclconvert
+
+## Overview
+
+## Dependencies
+
+* [bclconvert](https://emea.support.illumina.com/sequencing/sequencing_software/bcl-convert.html)
+
+
+## Usage
+
+### Cromwell
+```
+java -jar cromwell.jar run bclconvert.wdl --inputs inputs.json
+```
+
+### Inputs
+
+#### Required workflow parameters:
+Parameter|Value|Description
+---|---|---
+`runDirectory`|String|{'description': 'Illumina run directory (e.g. /path/to/191219_M00000_0001_000000000-ABCDE).', 'vidarr_type': 'directory'}
+`runName`|String|The name of the run, this will be used for the output folder and as a file prefix
```

Comment:
Is this redundant with runDirectory? (ie runName is the last level of the run directory path)

Reply:
Yes, it is the same run id

Reply:
Then why does one need to provide both?

Reply:
It was implemented like this originally, for the sake of convenience, I suppose. run id is used for report name. It may certainly be simplified